hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/ledger/accounts/staking/EndOfStakingPeriodCalculator.java

```diff
@@ -203,12 +203,18 @@ public void updateNodes(final Instant consensusTime) {
                 curNetworkCtx.pendingRewards(),
                 rewardsBalance());
 
+        final long reservedStakingRewards = curNetworkCtx.pendingRewards();
+        final long unreservedStakingRewardBalance = rewardsBalance() - reservedStakingRewards;
         final var syntheticNodeStakeUpdateTxn = syntheticTxnFactory.nodeStakeUpdate(
                 lastInstantOfPreviousPeriodFor(consensusTime),
                 nodeStakingInfos,
                 properties,
                 totalStakedRewardStart,
-                perHbarRate);
+                perHbarRate,
+                reservedStakingRewards,
+                unreservedStakingRewardBalance,
+                dynamicProperties.stakingRewardBalanceThreshold(),
+                dynamicProperties.maxStakeRewarded());
```

Comment:
Will these changes be made in the modularized code too?

Reply:
That's done [in this PR](https://github.com/hashgraph/hedera-services/blob/329e704688cc0acd6de7b4c3da68a498b826b2c4/hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/staking/EndOfStakingPeriodUpdater.java#L478). 😁